### PR TITLE
chore(curriculum): add JS functions review page

### PR DESCRIPTION
**Pull request by zairahira**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/review-javascript-functions/6723c5b8601a40a100bb59c9.md [hunk 1]
```diff
@@ -9,7 +9,28 @@ dashedName: review-javascript-functions
...
 
+## JavaScript Functions
 
+- Functions are reusable blocks of code that perform a specific task.
+- Functions can be defined using the `function` keyword followed by a name, a list of parameters, and a block of code that performs the task.
+- Arguments are values passed to a function when it is called.
+- When a function finishes its execution, it will always return a value.
+- By default the return value of a function is `undefined`.
+
+## Arrow functions
```
Comment (jdwilkin4) [inline curriculum/challenges/english/25-front-end-development/review-javascript-functions/6723c5b8601a40a100bb59c9.md:20]:
```suggestion
## Arrow Functions
```

File: curriculum/challenges/english/25-front-end-development/review-javascript-functions/6723c5b8601a40a100bb59c9.md [hunk 2]
```diff
@@ -9,7 +9,28 @@ dashedName: review-javascript-functions
...
+
+- Arrow functions are a more concise way to write functions in JavaScript.
+- Arrow functions are defined using the `=>` syntax between the parameters and the function body.
+- When defining an arrow function, you do not need the `function` keyword.
+- If you are using a single parameter, you can omit the parentheses around the parameter list.
+- If the function body consists of a single expression, you can omit the curly braces and the `return` keyword.
+
+## Scope in programming
```
Comment (jdwilkin4) [inline curriculum/challenges/english/25-front-end-development/review-javascript-functions/6723c5b8601a40a100bb59c9.md:28]:
```suggestion
## Scope in Programming
```

File: client/i18n/locales/english/intro.json [hunk 3]
```diff
@@ -2633,7 +2633,8 @@
       "review-javascript-functions": {
         "title": "JavaScript Functions Review",
         "intro": [
-          "Review the JavaScript Functions concepts to prepare for the upcoming quiz."
+          "Before you are quizzed on JavaScript functions, HTML, you first need to review the concepts.",
+          "Open up this page to review functions, arrow functions and scope"
```
Comment (jdwilkin4) [inline client/i18n/locales/english/intro.json:2637]:
```suggestion
          "Open up this page to review functions, arrow functions and scope."
```

**Review by jdwilkin4 — APPROVED** [review]
Applied a couple of small changes but otherwise good 👍

File: curriculum/challenges/english/25-front-end-development/review-javascript-functions/6723c5b8601a40a100bb59c9.md [hunk 4]
```diff
@@ -9,7 +9,28 @@ dashedName: review-javascript-functions
...
 Review the concepts below to prepare for the upcoming quiz.
 
+## JavaScript Functions
 
+- Functions are reusable blocks of code that perform a specific task.
+- Functions can be defined using the `function` keyword followed by a name, a list of parameters, and a block of code that performs the task.
+- Arguments are values passed to a function when it is called.
+- When a function finishes its execution, it will always return a value.
+- By default the return value of a function is `undefined`.
+
```
Comment (Dario-DC) [inline curriculum/challenges/english/25-front-end-development/review-javascript-functions/6723c5b8601a40a100bb59c9.md:19]:
```suggestion
- The `return` keyword is used to specify the value to be returned from the function and ends the function execution.

```

**Review by Dario-DC — APPROVED** [review]
Left a suggestion to add another bullet point. Feel free to adjust the sentence. Anyway LGTM :+1:

**Pull request merged**